src/Uno.UI/UI/Xaml/Media/GradientBrush.iOSmacOS.cs

```diff
@@ -5,6 +5,8 @@ namespace Windows.UI.Xaml.Media
 {
 	public abstract partial class GradientBrush : Brush
 	{
+		CALayer IGradientBrush.GetLayer(CGSize size) => GetLayer(size);
+
```

Comment:
Why an explicit interface implementation since both interface `IGradientBrush`  and the `GetLayer` method are `internal` ?
```suggestion
```

Reply:
@dr1rrb That's exactly the point. They are `internal`. C# requires implicit interface implementations to be public. Checkout [SharpLab](https://sharplab.io/#v2:EYLgtghglgdgNAExAagD6wC4FMBOMIA2ABJrgGYQDGWRAkgLABQA3k0e0QAIAsRAsgAoAlAG4mAXyZNOAZi4AmIgGEiIOk1aMOJGNjyEuvQULYdN2yY3FA==).